src/rpc/server/scala/api/src/main/scala/com/ctc/omega_edit/FFI.scala

```diff
@@ -109,6 +109,8 @@ private[omega_edit] trait FFI {
   def omega_session_pause_viewport_event_callbacks(p: Pointer): Unit
   def omega_session_resume_viewport_event_callbacks(p: Pointer): Unit
 
+    def omega_session_profile(p: Pointer, profile: Array[Long], offset: Long, length: Long): Int
```

Comment:
Indent needs fixed

Reply:
Yeah, my IntelliJ is very confused with Scala, but it does partially work.  It's mostly all red and the formatting is horrible, so I'm doing all the formatting by hand.